molecule/rbac-mds-plain-custom-rhel/molecule.yml

```diff
@@ -240,6 +242,17 @@ provisioner:
           - User:user1
 
       mds:
+
+        # necessary configs in MDS server for sso in C3
+        sso_groups_claim: groups
+        sso_sub_claim: sub
+        sso_issuer_url: http://ec2-35-166-19-61.us-west-2.compute.amazonaws.com:8080/auth/realms/cpsso
+        sso_jwks_uri: http://ec2-35-166-19-61.us-west-2.compute.amazonaws.com:8080/auth/realms/cpsso/protocol/openid-connect/certs
+        sso_authorize_uri: http://ec2-35-166-19-61.us-west-2.compute.amazonaws.com:8080/auth/realms/cpsso/protocol/openid-connect/auth
+        sso_token_uri: http://ec2-35-166-19-61.us-west-2.compute.amazonaws.com:8080/auth/realms/cpsso/protocol/openid-connect/token
+        sso_client_id: ssologin
```

Comment:
I see different values for sso clientid and password across different places, can we not re-use them ? 🤔

Reply:
We have 3 setups for testing sso. Okta, Azure and Keycloak. Each of these has a different id and password